app/src/main/java/org/piramalswasthya/sakhi/work/dynamicWoker/FormSyncWorker.kt

```
@@ -0,0 +1,68 @@
+
+            val unsyncedForms = repository.getUnsyncedForms()
+            for (form in unsyncedForms) {
+                if ((form.benId ?: -1) < 0) continue
+                val success = repository.syncFormToServer(form)
+                if (success) {
+                    repository.markFormAsSynced(form.id)
+                }
+            }
+
+            Result.success()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)